warehouse/templates/manage/organization/settings.html

```diff
@@ -225,20 +225,21 @@ <h3>{% trans %}Cannot delete organization{% endtrans %}</h3>
         {% pluralize %}
           Your organization currently owns {{ count }} projects.
         {% endtrans %}
-        {% trans count=active_projects|length %}
-          You must transfer ownership or delete this project before you can delete your organization.
-        {% pluralize %}
-          You must transfer ownership or delete these projects before you can delete your organization.
-        {% endtrans %}
+        {% trans %}For each project, you must either:{% endtrans %}
+        <ul>
+          <li>{% trans %}Transfer ownership and remove the project from your organization{% endtrans %}</li>
+          <li>{% trans %}Delete the project{% endtrans %}</li>
+        </ul>
+        {% trans %}Before you can delete your organization.{% endtrans %}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏non-blocking, but is there a way to support translators in that this sentence is part of the one preceding the list items?

Reply:
I pondered this as well, and tried

```
diff --git a/warehouse/locale/Makefile b/warehouse/locale/Makefile
index 844ea9f9e..6f511c2f3 100644
--- a/warehouse/locale/Makefile
+++ b/warehouse/locale/Makefile
@@ -5,6 +5,7 @@ compile-pot:
        PYTHONPATH=$(PWD) pybabel extract \
                -F babel.cfg \
                --omit-header \
+                --add-comments="i18n" \
                --output="warehouse/locale/messages.pot" \
                warehouse
 
diff --git a/warehouse/templates/manage/organization/settings.html b/warehouse/templates/manage/organization/settings.html
index d383b2c72..41d0e90f4 100644
--- a/warehouse/templates/manage/organization/settings.html
+++ b/warehouse/templates/manage/organization/settings.html
@@ -225,11 +225,13 @@
         {% pluralize %}
           Your organization currently owns {{ count }} projects.
         {% endtrans %}
+        {# i18n: Continued with "Before you can delete your organization." after a list of requirements #}
         {% trans %}For each project, you must either:{% endtrans %}
         <ul>
           <li>{% trans %}Transfer ownership and remove the project from your organization{% endtrans %}</li>
           <li>{% trans %}Delete the project{% endtrans %}</li>
         </ul>
+        {# i18n: Continuation of a sentence "For each project, you must either:" before a list of requirements #}
         {% trans %}Before you can delete your organization.{% endtrans %}
       </p>
       <ul class="no-bottom-margin">
```

But the comments weren't picked up.